utils/schemed-kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/schemedkafka/schemaregistry/universal/ParsedSchemaSupport.scala

```diff
@@ -114,7 +117,14 @@ class AvroSchemaSupport(kafkaConfig: KafkaConfig) extends ParsedSchemaSupport[Av
       rawParameter: Parameter,
       restrictedParamNames: Set[ParameterName]
   )(implicit nodeId: NodeId): ValidatedNel[ProcessCompilationError, SchemaBasedParameter] =
-    extractParameter(schema, rawMode, validationMode, rawParameter, restrictedParamNames)
+    extractParameter(
+      schema,
+      rawMode,
+      validationMode,
+      rawParameter,
+      restrictedParamNames,
+      AvroSchemaTypeDefinitionExtractorWithUnderlyingMap
```

Comment:
Let's add a comment why we use this version here.

Reply:
done